bokeh/_json_encoder.py

```diff
@@ -6,20 +6,12 @@
 import calendar
 import decimal
 
+from .util.dependencies import optional
 from .util.serialization import transform_series, transform_array
 import numpy as np
 
-try:
-    import pandas as pd
-    is_pandas = True
-except ImportError:
-    is_pandas = False
-
-try:
-    from dateutil.relativedelta import relativedelta
-    is_dateutil = True
-except ImportError:
-    is_dateutil = False
+pd = optional('pandas')
+dateutil = optional('dateutil')
```

Comment:
It looks like we require `dateutil` in `properties.py` can this just be converted to a regular import?